docs/manual-CN/server-administration/storage-metrics.md

```diff
@@ -0,0 +1,73 @@
+# Nebula  Storage Metrics
+## 介绍
+
+目前，Nebula 支持通过 HTTP 方式来获取 Storage Service 层操作的一些基本性能指标。
+
+每一个性能指标都由三部分组成，分别为指标名，统计类型，时间范围。<br />`<counter_name>.<statistic_type>.<time_range>`<br />下面将分别介绍这三部分。
```

Comment:
https://github.com/DavidAnson/markdownlint/blob/v0.17.0/doc/Rules.md#md033 best not html